Scenarioo/Scenarioo/Api/ScenarioDocuWriter.cs

```diff
@@ -234,7 +234,8 @@ public void SaveScreenshot(string usecaseName, string scenarioName, Step step, b
                 this.buildName, 
                 usecaseName, 
                 scenarioName, 
-                step.StepDescription.Index);
+                step.StepDescription.Index); // TODO: filename!
```

Comment:
@tobiaszuercher Can you please open an issue for this TODO, and reference the issue number in the TODO comment.
We have the rule, that a TODO comment should always belong to an issue in the backlog for managing and tracking them. 
Thanks a lot!